pkg/controllers/dashboard/systemcharts/controller_test.go

```diff
@@ -260,8 +258,8 @@ func Test_ChartInstallation(t *testing.T) {
 				mocks.manager.EXPECT().Ensure(
 					namespace.System,
 					"rancher-webhook",
-					"2.0.1",
 					"",
+					"2.0.4",
```

Comment:
This test should be deleted since it was created to test the deprecated settings. After these changes, it is the same test as above, with a different version number.